### PR TITLE
EA-759-sample-android-app-medicines

### PR DESCRIPTION
**Pull request by Arunkumarbwell**
Screenshot 2023-12-04 at 4 13 23 PM
Screenshot 2023-12-04 at 4 13 53 PM
Screenshot 2023-12-04 at 4 14 51 PM
Screenshot 2023-12-04 at 4 15 02 PM

### CODE REVIEW

**Comment by grantespo** [comment]
On the "What Is It" tab, I am not able to scroll all the way to the bottom, there is some cutoff text.

Screenshot 2023-12-04 at 3 18 44 PM

Also, should an image be displayed here?

Screenshot 2023-12-04 at 3 20 24 PM

File: bwell-kotlin-android/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -149,6 +149,12 @@
...
     <![CDATA[ 31/10/2023 \n   \n 0.3 mg/dl \n  \n Healthy Range \n  0.1 mg/dl-1.0 mg/dl \n   \n from Memorial health]]>
 </string>
-
+    <string name="active">Active</string>
+    <string name="past">Past</string>
+    <string name="medicine_detail">Medicine Detail</string>
+    <string name="overview">Overview</string>
+    <string name="waht_is_it">What is it ?</string>
+    <string name="pricing">About Pricing ?</string>
+    <string name="note_info">NOTE: Your data will be connected into your Lee Health app for your personal use to help you manage your healthcare. Lee Health will not review your health data from these connections and will only have access to it if you have consented to share this data with Lee Health.</string>
```
Comment (grantespo) [inline bwell-kotlin-android/app/src/main/res/values/strings.xml:158]: should these instances of "Lee Health" be removed from the value of `note_info`?

**Review by grantespo — APPROVED** [review]
left a couple of comments but approved.

**Comment by Arunkumarbwell** [comment]
> On the "What Is It" tab, I am not able to scroll all the way to the bottom, there is some cutoff text.
> 
> Screenshot 2023-12-04 at 3 18 44 PM
> Also, should an image be displayed here?
> 
> Screenshot 2023-12-04 at 3 20 24 PM

Done

**Review by istashak — APPROVED** [review]
Everything looks good as far as I can tell. The figma slides seem to be a little lacking around these screens, as far as I can tell.

**Pull request merged**